lib/ai_moderation/spam_scanner.rb

```diff
@@ -394,22 +394,27 @@ def self.handle_spam(post, log)
             queue_for_review: true,
           ).perform
 
-        log.update!(reviewable: result.reviewable)
-
-        reason = I18n.t("discourse_ai.spam_detection.silence_reason", url: url)
-        silencer =
-          UserSilencer.new(
-            post.user,
-            flagging_user,
-            message: :too_many_spam_flags,
-            post_id: post.id,
-            reason: reason,
-            keep_posts: true,
-          )
-        silencer.silence
+        # Currently in core re-flagging something that is already flagged as spam
+        # is not supported, long term we may want to support this but in the meantime
+        # we should not be silencing/hiding if the PostActionCreator fails.
+        if result.success?
```

Comment:
maybe we log something somewhere so we know it happened, do we have room in the ai table to add information that we tried to flag but it did not work?

Reply:
Good point, will see what I can add